src/appleseed-max-impl/appleseedrenderer/appleseedrenderer.cpp

```diff
@@ -165,6 +188,22 @@ void AppleseedRenderer::GetPlatformInformation(MSTR& info) const
 
 #endif
 
```

Comment:
I had to implemented Clone method in order to make interactive renderer actually start rendering.